src/app/create-message.ts

```diff
@@ -0,0 +1,3 @@
+export class CreateMessage {
```

Comment:
I'm trying to organize everything by "topic".
So a CreateMessage interface should live in the messaging folder.
Maybe something like messaging/model/ or messaging/dto/

Reply:
done.